net/third_party/quiche/src/quiche/common/platform/api/quiche_file_utils.cc

```diff
@@ -15,7 +15,18 @@ absl::optional<std::string> ReadFileContents(absl::string_view file) {
 bool EnumerateDirectory(absl::string_view path,
                         std::vector<std::string>& directories,
                         std::vector<std::string>& files) {
-  return EnumerateDirectoryImpl(path, directories, files);
+  std::vector<std::string> directories_full_path;
```

Comment:
Would it be possible for us to just rely on EnumerateDirectoryImpl and this is just the quicker way to fix things? If so, let's add a follow up bug

Either way, let's wrap this code in macros to make it clear where it's starboardized